src/grandorgue/model/GOEventHandlerList.h

```diff
@@ -40,9 +40,8 @@ class GOEventHandlerList {
   const std::vector<GOEventHandler *> &GetMidiEventHandlers() const {
     return m_MidiEventHandlers;
   }
-  const std::vector<GOPlaybackStateHandler *> &GetPlaybackStateHandlers()
-    const {
-    return m_PlaybackStateHandlers;
+  const std::vector<GOSoundStateHandler *> &GetPlaybackStateHandlers() const {
```

Comment:
Is is intentional to keep the old naming &GetPlaybackStateHandlers() for now?

Reply:
No? it isn't. I missed it.

Reply:
Fixed